docs/db.rst

```diff
@@ -0,0 +1,110 @@
+========================
+Freshmaker database
+========================
+
+Freshmaker as an SQL toolkit uses SQLAlchemy extension for `Flask <https://flask-sqlalchemy.palletsprojects.com/en/2.x/>`_. So whenever you want to add some columns or tables you should at least check ``models.py`` file. Every class that has a **FreshmakerBase** as a parent is actually a table with its own columns, relationships etc.
+
+As for database management system, we are using `PostgreSQL <https://www.postgresql.org/>`_ in deployments and `SQLite <https://www.sqlite.org/index.html>`_ in local environment.
+And besides that for testing we are using *in-memory* database, so whenever you are running tests from ``tests/`` folder, new database is created and live while tests are running.
+
+
+Setting up Database
+============================
+When you for the first time want to create freshmaker database, you are supposed to run:
+
+.. sourcecode:: none
+
+    ./manager createdb
+
+This command will create new ``freshmaker.db`` file for local use for you.
+
+
+Database update
```

Comment:
You're repeating these information below in the `Database Migrations` section, it might make sense to unify the sections and keep just one of the two.

Reply:
Yes it's repetitive, but *Database migration* section is more detailed and complex. From my view if I wanted to just make some changes in scheme(as specified in the task) I would like to know just a few steps/commands that are needed. And if I wanted to dig deeper into 'how migrations work' I would use that guide for migration.

Reply:
Alright then, in that case I'd say it looks good.